_our-population/Population Trends/marriage-parenthood.md

```diff
@@ -7,23 +7,25 @@ third_nav_title: Population Trends
+In addition, measures are in place to help parents manage their work and family commitments. These include the Tripartite Standards on Work-Life Harmony, Unpaid Leave for Unexpected Care Needs, and Flexible Work Arrangements (FWAs) established in 2021, 2018 and 2017 respectively. We are also currently working with tripartite partners to strengthen provisions on FWAs by introducing a new set of Tripartite Guidelines on FWAs by 2024, which will require employers to consider FWA requests from employees fairly and properly.
 
 More details on all the support measures can be found at [go.gov.sg/mpbooklet](https://go.gov.sg/mpbooklet) and [madeforfamilies.gov.sg](https://www.madeforfamilies.gov.sg/). Parents may also use the Family Support Calculator in the LifeSG app, to find out how much benefits they can receive.
-For parents to better manage their work and family responsibilities, we are working to cultivate more family-friendly workplaces.  This is encouraged through ongoing efforts such as the Tripartite Standards on Work-Life Harmony, Unpaid Leave for Unexpected Care Needs, and Flexible Work Arrangements (FWAs) established in 2021, 2018 and 2017 respectively. The Government will strengthen provisions on FWAs by introducing a new set of Tripartite Guidelines on FWAs by 2024. The Guidelines will require employers to consider FWA requests from employees fairly and properly.
 
-Beyond these, we must all do our part to ensure that Singapore is a great place to start and raise families. The Made For Families initiative was launched in June 2020 to further assure families of support from the Government and the broader society, and to encourage more businesses and community groups to play a part in building a family-friendly ecosystem. Organisations can submit their request to adopt the [Made for Families](https://www.madeforfamilies.gov.sg/) brand mark [here](https://www.madeforfamilies.gov.sg/about-us/made-for-families), to identify themselves as entities promoting the value of family in our society.
+Beyond these, we must all do our part to ensure that Singapore is a great place to start and raise families. The Made For Families initiative was launched in June 2020 to further assure families of support from the Government and the broader society, and to encourage more businesses and community groups to play a part in building a family-friendly ecosystem. Organisations can submit their request to adopt the [Made For Families](https://www.madeforfamilies.gov.sg/) brand mark [here](https://www.madeforfamilies.gov.sg/about-us/made-for-families), to identify themselves as entities promoting the value of family in our society.
```

Comment:
....building a family-friendly environment (to edit from ecosystem to environment)